collect_app/src/main/java/org/odk/collect/android/utilities/QRCodeUtils.java

```diff
@@ -55,9 +55,10 @@
 
 
 public class QRCodeUtils {
-    public static final String QR_CODE_FILEPATH = Collect.SETTINGS + File.separator + "collect-settings.jpeg";
-    static final String MD5_CACHE_PATH = Collect.SETTINGS + File.separator + ".md5";
+    public static final String QR_CODE_FILEPATH = Collect.SETTINGS + File.separator + "collect-settings.png";
```

Comment:
Is it enough to simply change the file extension to “png”?

Reply:
I am not sure. Let me check this